meson.build

```diff
@@ -23,8 +23,14 @@ config_h = configure_file(
   output: 'config.h',
 )
 
+drm_fourcc_header_path = join_paths(libdrm.get_pkgconfig_variable('includedir'), 'libdrm/drm_fourcc.h')
+
+tables_h = custom_target('tables_h',
+  output : 'tables.h',
+  command : [join_paths(meson.source_root(), 'fourcc.py'), drm_fourcc_header_path, '@OUTPUT@'])
+
 executable('drm_info',
-  files('main.c', 'json.c', 'pretty.c'),
+  [files('main.c', 'json.c', 'pretty.c'), tables_h],
```

Comment:
This nested syntax looks weird to me. Regardless, I don't think we actually need to add the header here. Meson will pick it up because it's `#include`'ed in the C file.

Reply:
It is required here. If I do not put it here and attempt to build after removing the build directory, I get `../pretty.c:13:10: fatal error: tables.h: No such file or directory`.

Reply:
Hmm, I see. It doesn't seem like you can put it in the `files()` list, can you?

Reply:
No, it must be the custom_target. Otherwise the header is simply not generated.